resolwe_bio/processes/import_data/sra_file.yml

```diff
@@ -156,26 +156,33 @@
     runtime: polyglot
     language: bash
     program: |
-
+      CACHE=$PWD/fastq_cache
```

Comment:
This is incorrect as `$PWD` may contain spaces. Please quote the string in double quotes.